jaeger_client/throttler.py

```diff
@@ -115,8 +115,7 @@ def callback():
         periodic = PeriodicCallback(
             callback=callback,
             # convert interval to milliseconds
-            callback_time=self.refresh_interval * 1000,
-            io_loop=self.channel.io_loop)
```

Comment:
please explain why `io_loop=self.channel.io_loop` is removed

Reply:
`io_loop` argument was deprecated and removed from Tornado 5+ (http://www.tornadoweb.org/en/stable/releases/v5.0.0.html#tornado-ioloop):

> The io_loop argument to PeriodicCallback has been removed.
> It is now possible to create a PeriodicCallback in one thread and start it in another without passing an explicit event loop.

But it's not obvious here, because `io_loop` that we are using here was made and started in another thread via `threadloop` (https://github.com/jaegertracing/jaeger-client-python/blob/master/jaeger_client/local_agent_net.py#L71) and we should pass the loop explicitly.
Fortunately we are starting polling through callback of the `io_loop` (https://github.com/jaegertracing/jaeger-client-python/blob/master/jaeger_client/throttler.py#L103 and https://github.com/jaegertracing/jaeger-client-python/blob/master/jaeger_client/sampler.py#L399) and the callback will be invoked in the thread of the `io_loop`. And that's why `PeriodicCallback` will take right loop to start (it takes existing loop from current thread), instead of creating new right in main thread.

Reply:
> argument was deprecated and removed from Tornado 5+ 

what about people still using it w/ Tornado 4?

Reply:
This argument is optional in Tornado 4: if no `io_loop` then use current loop https://github.com/tornadoweb/tornado/blob/branch4.5/tornado/ioloop.py#L998 Current loop will be taken from current thread and in our case it will be thread with already instantiated threadloop.